### PR TITLE
Drop start() and stop() in each qblox module

### PR DESCRIPTION
**Pull request by PiergiorgioButtarini**
Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

**Review by AleCandido — APPROVED** [review]
Now that #739 it would be useful to merge even this one.

Thanks @PiergiorgioButtarini for the PR!

**Comment by AleCandido** [comment]
Actually: I approved the PR just looking at the files changed (with seems alright), but there is some doctest failing (not sure why, since they are untouched and failing for reasons which do not seem strictly connected to this PR...)

**Comment by stavros11** [comment]
Doctest will be fixed by merging main here. There’s just a conflict to be addressed.

File: src/qibolab/instruments/qblox/cluster_qrm_rf.py [hunk 1]
```diff
@@ -1066,18 +1066,20 @@ def acquire(self):
...
-        try:
-            self.device.stop_sequencer()
-        except:
-            raise RuntimeError(f"Error stopping sequencer for {self.device.name}")
+        for sequencer_number in self._used_sequencers_numbers:
+            state = self.device.get_sequencer_state(sequencer_number)
+            if state.status != "STOPPED":
+                log.warning(
+                    f"Device {self.device.sequencers[sequencer_number].name} did not stop normally\nstate: {state}"
+                )
```
[the remark this answers is not in the document]
Comment (AleCandido) [reply src/qibolab/instruments/qblox/cluster_qrm_rf.py:1079]: #749

**Comment by stavros11** [comment]
@PiergiorgioButtarini thanks for the update. Could you also check the qpu tests before merging? There are some tests that are still calling `module.start()`/`module.stop()` which should probably be removed, but there is also a more crucial error
```
AttributeError: 'NoneType' object has no attribute 'get_sequencer_state'
```
This error also appears when calling `instrument.disconnect()` for qblox outside of tests (I tried running a qibocal routine and got the same error after execution finished).

**Comment by PiergiorgioButtarini** [comment]
@stavros11 now this PR should be ready to be merged. I removed the useless tests and fixed the error you spotted.

**Review by stavros11 — APPROVED** [review]
Thanks @PiergiorgioButtarini, it is fixed now.

**Pull request merged**